uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkDayTest.php

```diff
@@ -72,18 +72,12 @@ public function testTimeFromTimeToAndBreakShouldBeRequiredIfItsAWorkingDay($time
 
   /**
    * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidWorkDayException
-   * @expectedExceptionMessage Time From, Time To and Break should be empty for Non Working Days and Weekends
+   * @expectedExceptionMessage Break should be a valid number
    *
-   * @dataProvider timesAndBreakEmptyDataProvider
+   * @dataProvider invalidFormatBreakDataProvider
    */
-  public function testTimeFromTimeToAndBreakShouldBeEmptyIfItsNotAWorkingDay($timeTo, $timeFrom, $break) {
```

Comment:
Don't we need this test `testTimeFromTimeToAndBreakShouldBeEmptyIfItsNotAWorkingDay` again?
We still need a test that verify the exception message: `Time From, Time To, Break and Number of Hours should be empty for Non Working Days and Weekends`

Reply:
Done